pkg/machine/fcos.go

```diff
@@ -142,7 +177,28 @@ func getFCOSDownload(imageStream string) (*fcosDownloadInfo, error) {
 			logrus.Error(err)
 		}
 	}()
+	if imageStream == podmanTesting {
```

Comment:
I don't understand the need for this, the data we're getting should be...oh no, argh.  Right, there's 3 levels of JSON (cosa meta, release json, and stream JSON), and we actually only got the second.  I'm so sorry, I messed that up.

Well, it honestly doesn't matter - what matters is the binary can pull it, if we temporarily uglify this code I don't think it's a big deal.

For general context, Brent and I discussed this and I am firmly of the opinion we on CoreOS side (as well as elsewhere) should aim to ditch all our custom JSON long term in favor of OCI artifacts.  For more on that, see https://github.com/coreos/fedora-coreos-tracker/issues/828

Reply:
wait, this is not pretty? I  feel pretty.